examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/snarkjs/utils/computePublicInputs.ts

```diff
@@ -13,11 +12,11 @@ export const computeEffectiveEcdsaSigPublicInputs = (
   msgHash: Buffer,
   publicKeyHex: string,
 ) => {
+  const ec = new EC('secp256k1');
```

Comment:
@biscuitdey should we not make this configurable ... let's open an issue for that. PR is good for now.

Reply:
Will add to the issue for configurations (Milestone 5).